hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/common/HdfsServerConstants.java

```diff
@@ -368,6 +368,7 @@ public void write(DataOutput out) throws IOException {
       "system.hdfs.erasurecoding.policy";
 
   String XATTR_SATISFY_STORAGE_POLICY = "user.hdfs.sps";
+  String XATTR_SYNC_TO_EXTERNAL = "user.hdfs.sync";
```

Comment:
How is this XAttr used? I currently see this accessed only in `FSDirSyncNodeToRemoteStoreOp`